src/components/10-atoms/spinner/README.md

```diff
@@ -0,0 +1,30 @@
+# Spinner
+
+Adds a loading spinner
+
+```bash
+npm install @axa-ch/spinner
+```
+
+```js
+import '@axa-ch/spinner';
+...
+<axa-spinner></axa-spinner>
+```
+
+## Properties
+
+### Variant
+
+| Attribute                   | Details              |
+| --------------------------- | -------------------- |
+| `size=default`              | Size of the spinner  |
+| `color=inverted-blue-ocean` | Color of the spinner |
+
+### Size
+
+Declaires the size of the spinner.
+
+### Color
+
+Sets the color of the spinner. If `color` is set to `default`, it has the color ocean blue.
```

Comment:
If this is true, this should be refactored. Default-colors should not be named and `color="default"` does not make sense, since it is default. If you do not add the `color` attribute, you expect the default to happen. You don't expect, that you have to define attributes just to set them to defaults.